src/orders/fixtures/OrderFixture.ts

```diff
@@ -862,6 +863,227 @@ export class OrderFixture {
     return this;
   }
 
+  /**
+   * Adds multiple transactions with varied events for testing different scenarios:
+   * - Different event types (charge, refund, authorization)
+   * - Different creators (App, User, System/null)
+   * - Linked PSP references
+   */
+  withMultipleTransactions(): OrderFixture {
+    this.order = {
+      ...this.order,
+      transactions: [
+        // Transaction 1: Stripe with charge + refund flow (App creator)
+        {
+          ...OrderFixture.transaction,
+          id: "mock-transaction-1",
+          name: "Stripe",
+          pspReference: "pi_3ABC123",
+          externalUrl: "https://dashboard.stripe.com/payments/pi_3ABC123",
+          createdAt: "2024-12-01T10:00:00Z",
+          actions: [TransactionActionEnum.REFUND],
+          chargedAmount: { __typename: "Money", amount: 75.5, currency: "USD" },
+          refundedAmount: { __typename: "Money", amount: 25.0, currency: "USD" },
+          authorizedAmount: { __typename: "Money", amount: 0, currency: "USD" },
+          events: [
```

Comment:
issue: let's not create too large issue, because it can get hard to maintain them

I think we could instead add more to `OrderFixture.transaction` - e.g. different Transaction types, different events (or factories - aka functions to create such events based on parameters). Right now it's hard to read, due to all the required fields